enhancements/proxy-cache.md

```diff
@@ -0,0 +1,128 @@
+---
```

Comment:
Are these and the other non-update enhancement files attached to this pull request accidentally?  I don't see how they are relevant to the `mirror-registry-upgrade.md` proposal.